test/utils/configmaps.go

```diff
@@ -0,0 +1,17 @@
+package utils
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func ConfigMap(name, namespace string, labels, data map[string]string) *corev1.ConfigMap {
```

Comment:
This should be "NewConfigMap"  Do we not already have a configmap factory function in `pkg`